### PR TITLE
Further motivate the Broadmoor proposal using elevation data.

### PR DESCRIPTION
**Pull request by dabreegster**
There are 3 things we might show -- the contour map, the steep street coloring, or elevation profiles. I'm guessing we want to pick just one of them. Any opinions? And is this the place in the article where this information best fits?

@eldang, your opinion is more than welcome here of course. :)

### CODE REVIEW

**Comment by dabreegster** [comment]
I don't have an easy way to deploy the changes, so here's 3 screenshots at least. The 
Screenshot from 2021-07-07 14-41-26
contour map one is very vertical; if we go with it, maybe should size down.
Screenshot from 2021-07-07 14-41-30
Screenshot from 2021-07-07 14-41-14

**Comment by eldang** [comment]
Personally I find the elevation profiles the easiest to read, but it'll be more persuasive if you lock both Y axes to the same scale.  I think with that the decreased steepness and total climb will be intuitive from looking at the charts.

**Comment by dabreegster** [comment]
> but it'll be more persuasive if you lock both Y axes to the same scale.

This makes sense! The axis labels are a little too specific right now anyway. We have some code lying around for picking better intervals for axes; I'll apply it here and report back. Thanks!

**Comment by michaelkirk** [comment]
> Personally I find the elevation profiles the easiest to read, but it'll be more persuasive if you lock both Y axes to the same scale. I think with that the decreased steepness and total climb will be intuitive from looking at the charts.

Exactly what I was thinking!

**Comment by michaelkirk** [comment]
re: elevation-profiles.png 

Sounds like we're going to change this to adjust the axes anyway, but one more difference that we should address is the font.

- All the other graphics have the header above the image
- Preferably leave the header font out of the image so we can style the font / tweak layout with css/html.
- If you do bake the text into the image, use the overpass font

**Comment by dabreegster** [comment]
Should I use two separate images for the two elevation profiles? I just quickly squished them together and added the header in kolourpaint. Indeed styling with CSS might be easier.

**Comment by michaelkirk** [comment]
> Should I use two separate images for the two elevation profiles? 

Oh, I didn't realize they were one image. I haven't run it locally, but I imagine the image would be pretty small on mobile.

I'd recommend doing like with the "Time Saved / Lost" diagrams so that they are side-by-side on wides screens, but one on top of the other on mobile using the "wrap-two" class.

**Comment by dabreegster** [comment]
PTAL:
Screenshot from 2021-07-08 14-22-02

**Comment by eldang** [comment]
Nice!  This definitely improves the impact.  Can we also lock X axes to the same scale in both?  That will reinforce the distance message, and sidestep a possible grumble about steepness distortion.

**Comment by michaelkirk** [comment]
Getting better!

> Can we also lock X axes to the same scale in both? That will reinforce the distance message, and sidestep a possible grumble about steepness distortion.

Agreed that this would be nice! 

I think currently it's just two different screenshots - one taken before the proposal, and one taken after. I'm not sure how trivial it would be to plumb all that through. 

Using a consistent elevation was relatively easy because we used a map-wide constant to determine the max-Y, but I think we'll have to do some per-trip comparison for trip-length, so might be a relatively larger change.

I'd love to have a proper comparative elevation chart showing before vs. after in a single chart. Even with the uniform max-Y, it's a little hard to see... but that would be creeping our scope even more.

One quick compromise could be to, just for the purposes of this screenshot, hardcode the x-axis to (e.g.) 3 miles, and deliberately work on a proper comparative elevation chart for a future proposal.

## y-axis ticks

As you mentioned, having an odd number of labels for the y-axis produced awkward labels.

Personally I'd favor just two, so instead of `[0, 133.3, 266.7, 400]`, I think it would be more useful to have `[0, 200, 400]`. My understanding is that we're rounding the y-max to a power of ten, so dividing by 2 should always produce a nice round value.

Is it easy to remove a bucket? WDYT?

**Comment by dabreegster** [comment]
Screenshot from 2021-07-08 15-36-14
I just hardcoded the X-axis for the moment, as suggested. And lowered the number of buckets. Looking better?

**Comment by michaelkirk** [comment]
much better!

One final note: did you intentionally change trips or reverse the trip direction somehow?

**Comment by dabreegster** [comment]
I picked a different trip that goes in the opposite direction; agreed it looks a little funny. Let me pick a bit more carefully..

**Comment by dabreegster** [comment]
Screenshot from 2021-07-08 15-44-34
One more try

**Comment by eldang** [comment]
Nice!  This makes the strongest, easiest-to-follow case yet.

**Comment by michaelkirk** [comment]
I noticed the y-axis of the two charts, though they had the same units, didn't align vertically (just due to how the images were cropped)

I addressed this here if you'd like to merge them in: https://github.com/a-b-street/docs/compare/main...michaelkirk:mkirk/elevation_broadmoor?expand=1

But in the process, I had the two charts overlayed in krita anyway, and thought maybe this would be a better way to compare anyway, especially on mobile where the two charts won't be side by side:

elevation-profile

WDYT?

File: book/src/proposals/broadmoor.md [hunk 1]
```diff
@@ -594,4 +618,7 @@ what you'd like to see.
 
 [^sdot safety study]: Seattle Department of Transportation's safety study:
 [City of Seattle Bicycle and Pedestrian Safety Analysis Phase&nbsp;2](<https://www.seattle.gov/documents/Departments/SDOT/VisionZero/SDOT_Bike%20and%20Ped%20Safety%20Analysis_Ph2_2420(0).pdf>)
+
+[^elevation data]:
```
Comment (michaelkirk) [inline book/src/proposals/broadmoor.md]: Screen Shot 2021-07-08 at 4 38 15 PM


I guess the ordering here needs to reflect the ordering they appear in the article, so put elevation data first.

Comment (dabreegster) [reply book/src/proposals/broadmoor.md]: Good catch! Fixed

**Comment by dabreegster** [comment]
> I had the two charts overlayed in krita anyway, and thought maybe this would be a better way to compare anyway, especially on mobile where the two charts won't be side by side:

Thank you for the edits! Seeing the two lines together is the most clear, honestly. I think the text caption suffices to explain blue/green (along with the picture above it), so no need to squeeze in labels.

I think this looks pretty solid:
Screenshot from 2021-07-08 17-49-20

**Comment by eldang** [comment]
Agreed.  This feels like of discussion of 2 small charts, but I think each revision has successively told the story better.

**Review by michaelkirk — APPROVED** [review]
This LGTM, thanks for workshopping it with us!

**Pull request merged**